.github/workflows/ssh-command-example.yml

```diff
@@ -0,0 +1,55 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+on:
+  push:
+    branches:
+      - 'action-init'
+
+name: 'Connect to GCP instance via ssh and execute a command'
```

Comment:
Lets rename this to integration.yml if you were planning to use these as integration tests. If not lets add this to example-workflows repo (as this location is not discoverable)

Reply:
Have just removed the ymls, so will add intergation test in another PR separately